src/__mocks__/mock-data.js

```diff
@@ -0,0 +1,50 @@
+const data = {
+  users: [
+    {
+      id: 1,
+      firstName: `first`,
+      lastName: `last`,
+      username: `user1`,
+      password: `1234`,
+      role: `user`,
+      project: `project 1`,
+      position: `junior developer`,
+      imgUrl: `https://fishandconnorsawamovie.podbean.com/mf/web/6dhnad/deadpool_square2.jpg`,
+    },
+    {
+      id: 2,
+      firstName: `first`,
+      lastName: `last`,
+      username: `user2`,
+      password: `1234`,
+      role: `user`,
+      project: `project 2`,
+      position: 'senior developer',
+      imgUrl: `https://fishandconnorsawamovie.podbean.com/mf/web/6dhnad/deadpool_square2.jpg`,
```

Comment:
You might use `https://i.pravatar.cc/${size}` api to get random avatars :) e.g. https://i.pravatar.cc/300